File: src/test/java/leetcode/easy/array/ValidAnagram242Test.java
package leetcode.easy.array;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

class ValidAnagram242Test {

  private final ValidAnagram242 target = new ValidAnagram242();

  @Test
  public void test1() {
    assertTrue(target.isAnagram("anagram", "nagaram"));
  }

  @Test
  public void test2() {
    assertFalse(target.isAnagram("rat", "car"));
  }
}

File: src/test/java/leetcode/easy/hash/ValidAnagram242Test.kt
package leetcode.easy.hash

import org.junit.jupiter.api.Test
import kotlin.test.assertFalse
import kotlin.test.assertTrue

internal class ValidAnagram242Test {
    val target = ValidAnagram242()

    @Test
    fun test1() {
        assertTrue(target.isAnagram("anagram", "nagaram"))
    }

    @Test
    fun test2() {
        assertFalse(target.isAnagram("rat", "car"))
    }
}